src/outputs/output_utils.cpp

```diff
@@ -47,6 +47,11 @@ Triple_t<int> VarInfo::GetNumKJI(const IndexDomain domain) const {
     nx2 = std::max(nx2, cellbounds.ncellsj(domain, el));
     nx1 = std::max(nx1, cellbounds.ncellsi(domain, el));
   }
+  if (is_mem_aligned && DomainTouchesOuterGhosts(domain)) {
+    nx1 -= 1;
+    if (nx2 > 1) nx2 -= 1;
+    if (nx3 > 1) nx3 -= 1;
+  }
```

Comment:
Potentially general question: What happens to the outermost layer of face centered fields, i.e., the ones marking the domain boundary?
This is probably also related to the question of what the difference is (both internally and for outputs) between true face centered fields and face centered fields serving as fluxes for cells.

Reply:
Just to be clear, there are still outer ghosts, just one fewer. So if you had 2 you only have 1. That said I don't think the physical boundary code as ever been applied to `CellMemAligned` fields. That should probably be outside the purview of this MR.

Reply:
`CellMemAligned` just means that the underlying array has the same shape as a cell-centered field `(nz + 2 * nghost, ny + 2 * nghost, nx + 2 * nghost)` instead of `(nz + 2 * nghost + 1, ny + 2 * nghost + 1, nx + 2 * nghost + 1)`. For instance, that means that for a face-field in the x-direction, there is no storage for the last ghost face. This means we cannot set `Metadata::FillGhost` for `Metadata::CellMemAligned` fields. This is enforced at https://github.com/parthenon-hpc-lab/parthenon/blob/a0fdccf7007c5b83dec85d0deec5252b6c15882b/src/interface/metadata.cpp#L294